src/screens/Dashboard/Dushboard.models.ts

```diff
@@ -0,0 +1,9 @@
+export enum ViewMod {
```

Comment:
this name is too generic. I'd suggest renaming something like DashboardDataView.